lib/secure_headers/headers/content_security_policy.rb

```diff
@@ -219,13 +185,13 @@ def append_nonce(source_list, nonce)
       source_list
     end
 
-    # Private: return the list of directives that are supported by the user agent,
+    # Private: return the list of directives,
```

Comment:
Do you want to rewrap this line?

Reply:
Heh, I should probably take a pass over all of the comments. I'm sure there's plenty of outdated info.